app/views/foods/index.html.erb

```diff
@@ -0,0 +1,24 @@
+<section id="food">
+  <div class="btn-container">
+    <% if user_signed_in? %>
+      <%= button_to 'Add Food', '/foods/new', method: :get %>
+    <% end %>
+  </div>

```

Comment:
- [ ] When a user adds new food details, on submitting the form, they are redirected to the same form after submission. They should be redirected to the foods list instead.
- [ ] Also, on submission of the form, the foods list does not get updated i.e. the new entry does not appear. Give this a fix.